src/main/kotlin/no/nav/familie/ba/sak/kjerne/behandling/ReaktiverBehandlingPåVentService.kt

```diff
@@ -0,0 +1,83 @@
+package no.nav.familie.ba.sak.kjerne.behandling
+
+import no.nav.familie.ba.sak.kjerne.behandling.domene.Behandling
+import no.nav.familie.ba.sak.kjerne.behandling.domene.BehandlingRepository
+import no.nav.familie.ba.sak.kjerne.behandling.domene.BehandlingStatus
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+class ReaktiverBehandlingPåVentService(
+    private val behandlingRepository: BehandlingRepository,
+) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    /**
+     * @param åpenBehandlingId brukes for validering
+     * @param behandlingSomSniketIKøenId brukes for validering
+     */
+    fun reaktiverBehandlingPåVent(fagsakId: Long, åpenBehandlingId: Long, behandlingSomSniketIKøenId: Long) {
```

Comment:
Lurer litt på om det blir tydeligere om `åpenBehandlingId` burde døpes om til `behandlingPåVentId` / `åpenBehandlingPåVentId` 🤔 Og rename `behandlingSomSniketIKøenId` -> `behandlingSomSnekIKøenId`

```suggestion
    fun reaktiverBehandlingPåVent(fagsakId: Long, behandlingPåVentId: Long, behandlingSomSnekIKøenId: Long) {
```